inc/Classes/PasswordHash.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace LanSuite;
+
+class PasswordHash
```

Comment:
I do not favor making this class with a collection of static functions. I think we could benefit more from streamlining the scope of this class a bit more by e.g., removing the dependency on the configuration and injecting the configuration into the constructor.

This would make unit testing easier as well.

I can take this on and modify the class / this PR.